diff --git a/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js b/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
index 3405167..f41e3ef 100644
--- a/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
+++ b/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
@@ -14,7 +14,7 @@ const getUserAgent = (config) => {
             tertiary: config.version[2]
         },
         platform: config.browser[1].toLocaleLowerCase().includes('android')
-            ? proto.ClientPayload.UserAgent.Platform.ANDROID
+            ? proto.ClientPayload.UserAgent.Platform.SMB_ANDROID
             : proto.ClientPayload.UserAgent.Platform.WEB,
         releaseChannel: proto.ClientPayload.UserAgent.ReleaseChannel.RELEASE,
         osVersion: '0.1',
